apps/price_pusher/src/solana/solana.ts

```diff
@@ -93,10 +93,11 @@ export class SolanaPricePusher implements IPricePusher {
       return;
     }
 
+    const shuffledPriceIds = priceIds.sort(() => Math.random() - 0.5);
```

Comment:
Hmmmm is this key computed only once? :? I'm wondering if it would crash if it's recalibrated each time

Reply:
I don't understand your comment